test/task_test.py

```diff
@@ -211,12 +209,9 @@ def run(self):
 
     def test_externalize_same_id_with_task_namespace(self):
         # Dependent on the new behavior from spotify/luigi#1953
-        class MyTask(luigi.Task):
+        class MyTask(RunOnceTask):
             task_namespace = "something.domething"
 
-            def run(self):
```

Comment:
Again this isn't equivalent and you didn't need to change this...

However in this case I guess we have a more defined task so this is okay too.